client/src/styles/BkmarkButton.css

```diff
@@ -0,0 +1,3 @@
+body {
+    background-color: #FDF6F0;
+}
```

Comment:
Per issue #105, please move all instances of `body` and `:root` into the `index.css` file.